action.yml

```diff
@@ -86,20 +86,20 @@ runs:
         python-version: ${{ inputs.python-version }}
         architecture: x64
         cache: pip
-    
-    - run: python --version
-      shell: bash
 
     - name: Windows install dependencies
       if: ${{ runner.os == 'Windows' }}
       run:  |
-        pip install -r ${{ github.action_path }}\requirements.txt
+        pip install -r requirements.txt  # project's dependencies
+        pip install -r ${{ github.action_path }}\requirements.txt  # linters
         echo "path_sep=" >> $GITHUB_ENV
       shell: pwsh
 
     - name: Posix install dependencies
       if: ${{ runner.os != 'Windows' }}
-      run:  pip install -r ${{ github.action_path }}/requirements.txt
+      run:  |
+        pip install -r requirements.txt  # project's dependencies
+        pip install -r ${{ github.action_path }}/requirements.txt  # linters
```

Comment:
`setup-python`'s `cache` argument was expecting a requirements.txt but is not doing the install

See the example in their readme file:
https://github.com/actions/setup-python#caching-packages-dependencies

Reply:
yeah, i have noticed this as well. Thanks for the fix